apps/Standalone/src/templates/app/TemplatesStandaloneDesigner.tsx

```diff
@@ -190,69 +190,75 @@ export const TemplatesStandaloneDesigner = () => {
             isConsumption={isConsumption}
             existingWorkflowName={existingWorkflowName}
           >
-            <TemplateFilters
-              connectors={[
-                {
-                  value: 'azureaisearch',
-                  displayName: 'Azure AI Search',
-                },
-                {
-                  value: 'openai',
-                  displayName: 'Open AI',
-                },
-                {
-                  value: 'sql',
-                  displayName: 'SQL',
-                },
-                {
-                  value: 'amazon',
-                  displayName: 'Amazon',
-                },
-              ]}
-              detailFilters={{
-                Trigger: [
-                  {
-                    value: 'Request',
-                    displayName: 'Request',
-                  },
-                  {
-                    value: 'Instant',
-                    displayName: 'Instant',
-                  },
-                ],
-                By: [
-                  {
-                    value: 'Microsoft',
-                    displayName: 'Microsoft',
-                  },
-                  {
-                    value: 'Other',
-                    displayName: 'Other',
-                  },
-                ],
-                Type: [
+            <div
+              style={{
+                margin: '20px',
+              }}
+            >
+              <TemplateFilters
+                connectors={[
                   {
-                    value: 'Workflow',
-                    displayName: 'Workflow',
+                    value: 'azureaisearch',
+                    displayName: 'Azure AI Search',
```

Comment:
While this is fine for now, I think we should figure out a way to make these not hard coded. It’s not going to scale

Reply:
This is coming in my next PR